include/deal.II/base/mpi_consensus_algorithms.h

```diff
@@ -440,6 +445,34 @@ namespace Utilities
         clean_up_and_end_communication();
       };
 
+      /**
+       * A serial fall back for the above classes to allow programming
+       * independently if MPI is used or not.
+       */
+      template <typename T1, typename T2>
+      class Serial : public Interface<T1, T2>
+      {
+      public:
+        /**
+         * Constructor.
+         *
+         * @param process Process to be run during consensus algorithm.
+         * @param comm MPI Communicator (ignored)
+         */
+        Serial(Process<T1, T2> &process, const MPI_Comm &comm);
+
+        /**
+         * Destructor.
+         */
+        virtual ~Serial() = default;
```

Comment:
Do we need this constructor? I believe the virtual destructor in the base class `Interface` should be enough.

Reply:
I have removed. Let's see if one of the compilers complain...